zerver/views/report.py

```diff
@@ -78,7 +80,7 @@ def json_report_error(request, user_profile, message=REQ(), stacktrace=REQ(),
                       ui_message=REQ(validator=check_bool), user_agent=REQ(),
                       href=REQ(), log=REQ(),
                       more_info=REQ(validator=check_dict([]), default=None)):
-    # type: (HttpRequest, UserProfile, str, unicode, bool, str, str, str, Dict[str, Any]) -> HttpResponse
```

Comment:
I think the following should be typed as `text_type` instead of `str`:
`message`, `user_agent`, `href`, `log`.